janusgraph-hbase-parent/janusgraph-hbase-core/src/test/java/org/janusgraph/diskstorage/hbase/HBaseStoreManagerMutationTest.java

```diff
@@ -127,4 +127,41 @@ public void testKCVMutationToPuts() throws Exception {
         Collections.sort(deleteColumns);
         Assert.assertArrayEquals(expectedColumnDelete.toArray(), deleteColumns.toArray());
     }
+
+    @Test
+    public void testMutationToPutsTTL() throws Exception{
+        final Map<String, Map<StaticBuffer, KCVMutation>> storeMutationMap = new HashMap<>();
+        final Map<StaticBuffer, KCVMutation> rowkeyMutationMap = new HashMap<>();
+        final List<Long> expectedColumnsWithTTL = new ArrayList<>();
+        final List<Long> putColumnsWithTTL = new ArrayList<>();
+        List<Entry> additions = new ArrayList<>();
+        List<StaticBuffer> deletions = new ArrayList<>();
+        StaticArrayEntry e;
```

Comment:
We can move `StaticArrayEntry e;` to line 145. Then it will be initialized like:
`StaticArrayEntry e = (StaticArrayEntry) StaticArrayEntry.of(col, val);`